build/media-suite_helper.sh

```diff
@@ -769,7 +769,7 @@ do_changeFFmpegConfig() {
     eval "$(sed -n '/EXTERNAL_LIBRARY_VERSION3_LIST=/,/^"/p' "$config_script")"
 
     # handle gpl libs
-    local gpl=(${EXTERNAL_LIBRARY_GPL_LIST//_/-} gpl)
+    local gpl=("${EXTERNAL_LIBRARY_GPL_LIST//_/-}" gpl)
```

Comment:
Doesn't this make all the non-gpl libs into a single entry in the array?

Reply:
Need to check again for that, probably yes, need to fix

Reply:
Found a way to use read and properly set the arrays. Going to rerun the suite to test. I sort of wish there was a way to quickly run the suite with all configs to test.